create_llamaedge_sysext.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export ARCH="${ARCH-x86-64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download the llamaedge release tar ball (e.g., for 0.14.16) and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "To use arm64 pass 'ARCH=arm64' as environment variable (current value is '${ARCH}')."
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+# The github release uses different arch identifiers, we map them here
+# and rely on bake.sh to map them back to what systemd expects
+if [ "${ARCH}" = "amd64" ] || [ "${ARCH}" = "x86-64" ]; then
+  ARCH="x86_64"
+elif [ "${ARCH}" = "arm64" ]; then
+  ARCH="aarch64"
+fi
+
+# llamaedge is a wasm application, which requires WasmEdge and its WASI-NN GGML/GGUF plugin
```

Comment:
Sure, let me create a local image and test it. I will update this PR once the verification is done. Thanks for your information.

Also, I have a question: the plugin, `liblibwasmedgePluginWasiNN.so`, has multiple versions. This PR ships the pure CPU version. We also have two versions: one for CUDA-11 and another for CUDA-12. Could you please give me some hints on how to handle this case?

Reply:
I have separated the original llamaedge.raw and now rely on the wasmedge sysext image. I also verified it on the Digital Ocean instances.

Reply:
Hi @tormath1,  
Since I don't have an environment to verify the GPU-enabled behavior, I will resolve this conversation and make this PR provide only the pure CPU version of LlamaEdge. Thanks.